src/events/eventActions.js

```diff
@@ -141,7 +71,7 @@ const startEventLoop = (auth, queueId, eventId) =>
     }
   };
 
-export const fetchEvents = (auth: Auth) =>
+export const fetchEvents = (auth: Auth, state) =>
```

Comment:
Good catch. I added it before realizing the action creator gets a receives a getState anyways.